covalent/executor/executor_plugins/remote_executor.py

```diff
@@ -0,0 +1,164 @@
+# Copyright 2021 Agnostiq Inc.
+#
+# This file is part of Covalent.
+#
+# Licensed under the GNU Affero General Public License 3.0 (the "License").
+# A copy of the License may be obtained with this software package or at
+#
+#      https://www.gnu.org/licenses/agpl-3.0.en.html
+#
+# Use of this file is prohibited except in compliance with the License. Any
+# modifications or derivative works of this file must retain this copyright
+# notice, and modified files must contain a notice indicating that they have
+# been altered from the originals.
+#
+# Covalent is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the License for more details.
+#
+# Relief from the License may be granted by purchasing a commercial license.
+
+"""
+Module for defining a Dask executor that submits the input python function in a dask cluster
+and waits for execution to finish then returns the result.
+
+This is a plugin executor module; it is loaded if found and properly structured.
```

Comment:
Let's try to improve this docstring. Break that first part into a few simple, complete sentences. Explain more about what "found and properly structured" means.

Reply:
I have update the docstring following these points.